Source/LinqToDB.EntityFrameworkCore/LinqToDBForEFTools.cs

```diff
@@ -558,5 +566,27 @@ public static bool EnableChangeTracker
 			set => Implementation.EnableChangeTracker = value;
 		}
 
+		private static void AddDefaultInterceptorsToDataContext(IDataContext dc)
+		{
+			if (dc != null && Implementation.DefaultLinq2DbInterceptors?.Any() == true)

```

Comment:
If `DefaultLinq2DbInterceptors` is no longer settable, then you have confidence the value always exists.
```suggestion
			if (dc != null && Implementation.DefaultLinq2DbInterceptors.Any())
```

Reply:
I agree, Adjusted.